packages/site/src/data/nodejs/libraries.ts

```diff
@@ -1,26 +1,166 @@
 import { Library } from "@framework/system/src/models/library"
 
 export const libraryTags = [
-	"state management",
-	"internationalization",
 	"framework",
 	"documentation",
-	"styling",
-	"component",
 	"logging",
```

Comment:
When tend to use `authentication` for this tag

Reply:
Do you mean `authentication` in place of logging?

I can add that as a tag, but there aren't specific node libs that focus on authentication. And there are libs specifically for logging in node (planning to add more libs featuring those in another PR).

Reply:
Oh! then `logging` is fine 🚀